### DISCUSSION TITLE
Support for editing HAProxy maps

### DISCUSSION THREAD

**Discussion by spiglee** [roxy-wi/roxy-wi#333] [General]
I may be missing it from all of my testing on the demo site and reading the documentation / howto documents, but I don't readily see a way to edit and maintain maps with with the web interface. Is there support for managing HAProxy maps? If not, are there plans to add support?

https://www.haproxy.com/blog/introduction-to-haproxy-maps/

**Comment by Aidaho12** [comment]
There is no a way to edit maps now, but if you would like to have, we can implement it

**Comment by spiglee** [comment]
Thanks for the information. As I have not committed to installing and using Roxy-WI at this time, I wouldn't feel right asking for a feature for something we haven't committed to using yet. We won't be able to move to using this without this feature as updating the maps is as important as updating the haproxy.cfg file given we manage hundreds of URLs in the maps file that change as regularly as the list of backend servers changes in the haproxy.cfg. We'll continue to manage the HAProxy cluster through something like Ansible.

**Reply by Aidaho12** [reply]
I guess it useful feature even you won't be use Roxy-WI, but I hope you will with time :)
